src/website/webview/views.py

```diff
@@ -492,8 +492,18 @@ class SuggestionListView(ListView):
     def get_context_data(self, **kwargs: Any) -> Any:
         context = super().get_context_data(**kwargs)
 
+        affected_pk = list()
```

Comment:
With https://github.com/Nix-Security-WG/nix-security-tracker/pull/369 you can now do

```suggestion
        affected_pk = list()
        metrics_pk = list()
        
        for obj in context["object_list"]:
            metrics_pk.extend(obj.cve.container.values_list("metrics", flat=True))
        metrics = Metric.objects.in_bulk(metrics_pk)
```

The only difference we have to figure out is whether we want *all* the `baseSeverity`s from the list of `Metric`s or e.g. a "max" over the enum.